libs/src/api/playlist.js

```diff
@@ -206,18 +207,37 @@ class Playlists extends Base {
     }
   }
 
+  /**
+   * Uploads a cover photo for a playlist without updating the actual playlist
+   * @param {number} playlistId
+   * @param {File} coverPhotoFile the file to upload as the cover photo
+   * @return {string} CID of the uploaded cover photo
+   */
+  async uploadPlaylistCoverPhoto (playlistId, coverPhotoFile) {
```

Comment:
looks like playlistId isn't used. also do we need the onProgress and timeoutMs values?

Reply:
you're right -- it's not. fixed!